pkg/virt-handler/vm.go

```diff
@@ -1661,13 +1664,16 @@ func (c *VirtualMachineController) Run(threadiness int, stopCh chan struct{}) {
 
 	heartBeatDone := c.heartBeat.Run(c.heartBeatInterval, stopCh)
 
+	c.multipathSocketMonitor.Run()
```

Comment:
@0xFelix Why are we not passing the stopCh to this monitor as well?

Reply:
I thought about it, but I had the feeling that it would make the monitor's Run/Close operations and the associated code unnecessarily difficult to read/understand. However, I can adapt it if you want.

Reply:
I think I'm with Vladik on this